.github/workflows/release-srtool.yml

```diff
@@ -7,7 +7,7 @@ env:
 on:
   push:
     tags:
-      - "*"
+      - "release-*" # TODO what to put here?
```

Comment:
@EgorPopelyaev dont know why it runs on *any* tag? Should it?

Reply:
Good question, maybe it was done for some testing purposes, to built runtimes and test those. But for this case, could be used the manual flow from the [srtool repo itself](https://github.com/paritytech/srtool/actions/workflows/manual-polkadot-sdk.yml). For the release, we don't need to use tags, because this flow is triggered either from the one which [creates a release draft](https://github.com/paritytech/polkadot-sdk/blob/935c7f461ae8b4e607f1db16322ea952b438650e/.github/workflows/release-30_publish_release_draft.yml#L26) or by push to the branch directly

Reply:
Then i will remove the tags trigger.